langchain/utils.py

```diff
@@ -111,3 +111,14 @@ def now(cls):  # type: ignore
         yield datetime.datetime
     finally:
         datetime.datetime = real_datetime
+
```

Comment:
@dev2049 do we have a better namespace for this in langchain? This utility does not seem like it will be commonly used enough to end up in `langchain.utils.py`